frontend/src/services/GooglePlacesService.ts

```diff
@@ -1,33 +1,34 @@
-import axios, { AxiosError, AxiosResponse } from 'axios';
-import * as dotenv from 'dotenv';
+import axios, { AxiosError, AxiosResponse } from "axios";
+import * as dotenv from "dotenv";
 
 /* eslint-disable  no-console */
 // todo: remove this eslint-disable
 class GooglePlacesService {
+  public currRestaurants: Map<string, Record<string, unknown>>;
+
   constructor() {
     dotenv.config();
+    this.currRestaurants = new Map();
   }
 
-  getGooglePlacesSearch(data: string): Promise<JSON> {
```

Comment:
So will this data string contain both the restaurant name and it's address? Because I see you're taking the first candidate, but would we also need the restaurant specific to the address/location.

Reply:
Yup, I didn't add it into my description here but I was planning to address handling the edge case for franchises for this week's task! I'm still figuring out how to best handle this case.

Reply:
Ooh okie got it!